### PR TITLE
Added BlogPage fields validation

### LINKED ISSUE #103: Add validation to BlogPage fields in information structure
Collection name: `"blog"`
Task description in linked Epic issue.
https://github.com/CodersCrew/CodersCard/issues/98#issue-725735917

### PR DESCRIPTION
**Pull request by celinawierzbicka**
Closes #103

### CODE REVIEW

**Review by mecies — CHANGES_REQUESTED** [review]
Nice job @celinawierzbicka 👏 
I have one suggestion and some comments, please see below 😄 

In `config.yml`, line 20, please add validation for the page title, I can suggest setting a gap at 50 or 60 characters

source:
https://moz.com/learn/seo/title-tag#:~:text=Optimal%20title%20length,your%20titles%20to%20display%20properly.

File: static/admin/config.yml [hunk 1]
```diff
@@ -23,11 +23,11 @@ collections:
         widget: "list"
         required: false
         fields:
-          - { label: "Title", name: "blogTitle", widget: "string"}
+          - { label: "Title", name: "blogTitle", widget: "string", pattern: ['.{5,60}', "Must have 5-60 characters"]}
           - { label: "Publish Date", name: "publishDate", widget: "datetime" }
-          - { label: "Label", name: "blogLabel", widget: "string" }
-          - { label: "Description", name: "blogDescription", widget: "string" }
-          - { label: "Body", name: "blogBody", widget: "markdown" }
+          - { label: "Label", name: "blogLabel", widget: "string", pattern: ['.{3,20}', "Must have 3-20 characters"]} }
```
Comment (mecies) [inline static/admin/config.yml]:
```suggestion
          - { label: "Label", name: "blogLabel", widget: "string", pattern: ['.{3,20}', "Must have 3-20 characters"]}
```
please remove the additional bracket, or the project will not compile 🗒️

File: static/admin/config.yml [hunk 2]
```diff
@@ -23,11 +23,11 @@ collections:
...
+          - { label: "Description", name: "blogDescription", widget: "string", ['.{20,250}', "Must have 20-250 characters"]}
+          - { label: "Body", name: "blogBody", widget: "markdown", ['.{100,3000}', "Must have 100-3000 characters"] }
```
Comment (mecies) [inline static/admin/config.yml]:
```suggestion
          - { label: "Description", name: "blogDescription", widget: "string", pattern: ['.{20,250}', "Must have 20-250 characters"]}
          - { label: "Body", name: "blogBody", widget: "markdown", pattern: ['.{100,3000}', "Must have 100-3000 characters"] }
```
add missing pattern keywords 😄

Comment (celinawierzbicka) [reply static/admin/config.yml]: Fixed!

File: static/admin/config.yml (hunk 1 above)
Comment (celinawierzbicka) [reply static/admin/config.yml]: fixed

**Review by mecies — APPROVED** [review]
Nice job @celinawierzbicka , ready to merge ✅

**Pull request merged**